src/main/java/pro/sky/homeworkemployee/Employee.java

```diff
@@ -0,0 +1,44 @@
+package pro.sky.homeworkemployee;
+
+import java.util.Objects;
+
+public class Employee {
+
+    private String name;
+    private String surName;
```

Comment:
```suggestion
    private final String name;
    private final String surName;
```
поскольку поле не изменяются по коду